cli/commands.go

```diff
@@ -142,11 +138,15 @@ func CreateCluster(c *cli.Context) error {
 		return err
 	}
 
+	if c.IsSet("timeout") {
+		log.Println("[Warning] The --timeout flag is deprecated. use '--wait <timeout>' instead")
```

Comment:
Don't we have to do a `log.Fatalln` here, since the `--timeout X` wouldn't have any effect here, as we get the value from the `--wait` flag now?

Reply:
Sure, I can make the change.